components/eam/cime_config/testdefs/testmods_dirs/eam/nlmaps/nlmaps_check.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
```

Comment:
We don't have specs or style enforcers for python-in-the-code base because...we didn't expect it.   Usually the python-related code is in CIME which does have those kind of checks and standards.

Reply:
Yes, I think I'm the only user of the excellent POSTRUN_SCRIPT testing feature.